src/components/view-more/view-more.scss

```diff
@@ -0,0 +1,60 @@
+.view-more-panel-body {
+	max-height: 70vh;
+	position: relative;
+	transition: max-height 0.3s cubic-bezier(0.76, 0, 0.24, 1);
+}
+
+.view-more-panel-cta {
+	background: transparent;
+	border: 0;
+	color: currentColor;
+	cursor: pointer;
+	display: none;
+	font-size: 15px;
+	font-weight: 600;
+	margin: 0 auto;
+
+	.downward-arrow {
+		fill: $color-grey-700;
```

Comment:
could you use `@use` syntax here and import `colors` ? ditto for $bp-500 / any variables

trying to migrate to `@use` so hoping all new work adopts it

Reply:
Sure. let's do